papers/project_jupyter/paper.rst

```diff
@@ -561,15 +608,56 @@ in Python <https://sphinx-gallery.readthedocs.io/en/latest/advanced_configuratio
 with Sphinx Gallery, and sharing `interactive content <http://greenteapress.com/wp/think-dsp/>`_
 that requires a language-specific kernel in order to run.
 
+Cost and sustainability  for ``mybinder.org``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 Mybinder.org is currently funded through a grant from the Moore foundation, and
 the team is actively exploring new models for keeping the service running sustainably.
 The public service currently restricts the hardware available to users in several
 ways in order to keep costs down. For example, users are only given access to one
 CPU, two gigabytes of RAM, can only access public git repositories, and are
-restricted in the kinds of network I/O that can take place. The Binder team hopes
-that other organizations, companies, or universities will deploy their own
-BinderHubs for their own users, potentially powered by more complex hardware
-or more permissive environments.
+restricted in the kinds of network I/O that can take place. It currently
+runs on the Google Cloud Platform (though could be run on any setup that runs
+on top of Kubernetes).
+
+.. figure:: images/cost_breakdown.png
+   :align: center
+
+   Cloud computing costs for running ``mybinder.org`` in 2018. x-axis shows
+   one point per day. Daily unique users has consistently grown over this time,
+   while modifications to the BinderHub codebase (as well as the cloud
+   resources used) has kept costs relatively flat. As a result, ``mybinder.org``
+   currently operates at about 5 cents per user per day.
+
+The decision to avoid the notion of a user "identity" in particular has strong
+effects on the cost of running a BinderHub server. Because users do not require
+persistent storage (e.g. the content of any changes they make to Jupyter
+Notebooks throughout a session), a significant cost of running a JupyterHub
+is avoided. In addition, because Kubernetes is built with flexibiliy in mind,
+a BinderHub deployment can efficiently use the resources available to it in
+order to avoid over-provisioning cloud resources as much as possible.
+
+Currently, ``mybinder.org`` runs at a cost of several hundred dollars per
+week. At roughly 50,000 users per week, this comes out to around $0.05 per
+user. The ``mybinder.org`` team publishes its daily costs in `a public
+repository on GitHub <https://github.com/binder-examples/binder-billing>`_.
+It hopes that this serves to encourage other organizations to deploy BinderHub
+for their own purposes, since it is possible to do so in a cost-effective
+manner.
+
+Moving forward, the Binder team is interested in exploring multiple models
+for sustaining the public ``mybinder.org`` deployment, as well as the
+broader Binder ecosystem. First, at its current rate the annual cost of running
+``mybinder.org`` is around $80,000, an amount that could be sustainable with
```

Comment:
I would remove this paragraph or make it much more vague "We are currently exploring all possible options for making the Binder project sustainable: grant funding, commercial options, federation, 'friends of binder', etc". It will be hard to put a realistic number for the running cost here. So I would definitely not list something like $80,000. Running mybinder.org reliably requires several full time people who need to be paid so that cost needs to be included if we want to give out numbers.

For my curiosity, how did you arrive at the 80k number? "several hundred dollar per week" * 52 is only around $40k

Reply:
That's fair - could you open an issue to discuss there? (my bad for merging this PR pre-emptively)

re: 80K, we're spending around $250 a day, which amounts to about 80-90K a week. I should update the wording in the text to be more precise. Also see https://github.com/choldgraf/scipy_2018_binder which shows all of our calculations

Reply:
Where should we put issues like this? This repo or mybinder.org-deploy or ...? If this repo could you use your admin powers to enable issues :D ?

Reply:
oops, I forgot issues aren't enabled on a forked repo. they are enabled now!